### PR TITLE
Issue200 - Remove obsolete "Staking Only" feature flag

### PR DESCRIPTION
**Pull request by Nizzypal**
### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

**References  AND even definitions have been removed.  PLEASE NOTE: Removed flag was involved as one of  the paths in several  control-flow logic.**

### PR Description

**Issue200 - Remove obsolete "Staking Only" feature flag**

### CODE REVIEW

**Comment by seiyria** [comment]
You should also remove references to featureFlagStake etc. You can't just remove them in these places.

File: frontend/src/router.ts [hunk 1]
```diff
@@ -12,21 +12,20 @@ import Portal from './views/Portal.vue';
...
-  if (featureFlagStakeOnly) {
-    return new VueRouter({
-      routes: [
-        { path: '/', redirect: 'stake' },
-        { path: '/stake', name: 'select-stake-type', component: SelectStakeType },
-        { path: '/stake/:stakeType', name: 'stake', component: Stake, props: true },
-      ]
-    });
-  }
+  // if (featureFlagStakeOnly) {
```
Comment (seiyria) [inline frontend/src/router.ts:20]: the reason we're intending to remove it is not to leave it as comments in the codebase. please remove the code entirely.

**Review by seiyria — CHANGES_REQUESTED** [review]

File: frontend/src/router.ts (hunk 1 above)
Comment (Nizzypal) [reply frontend/src/router.ts:20]: Ok, I just thought you guys might want some substitute logic for the logic flows that this flag catches... guess  I can delete them safely then

Comment (Nizzypal) [reply frontend/src/router.ts:20]: Btw in env.mainnet and testnet.. do I delet the flag?  Coz it says just permanently disabled so Im unsure?

VUE_APP_STAKING_ONLY="0"---- > delete?

**Comment by seiyria** [comment]
yes, we would like to remove all traces of it.

**Comment by Nizzypal** [comment]
Generally we were  taught to comment large blocks of code that we didnt write instead of  deleting them esp if we didnt write it, and generally ask for a review before nuking it :)

**Comment by seiyria** [comment]
No, I'm not sure who taught you that but we have version control for that. I can see the lines prefaced with `-` and it's highlighted red. Commented code just takes up a lot of space that it doesn't need to.

**Comment by Nizzypal** [comment]
No you  misunderstand.  Its not to let it remain there.  It just needs a reviewbefore deleting.  And in this case you asking for it to be removed from comments is that review.

**Comment by Nizzypal** [comment]
Version contrl  goes without  saying, itsbeen a thing for more  than a decade my friend

**Comment by Nizzypal** [comment]
made the link in the  description bigger & bolder for ya.

**Comment by raymond-h** [comment]
No, he understood perfectly fine, and his point still stands: you need not worry about leaving existing code as comments - the Pull Request UI in Github will show us what was removed, so we can tell from there what was deleted. In fact, leaving them as comments just makes the diff bloated.

**Comment by Nizzypal** [comment]
> No, he understood perfectly fine, and his point still stands: you need not worry about leaving existing code as comments - the Pull Request UI in Github will show us what was removed, so we can tell from there what was deleted. In fact, leaving them as comments just makes the diff bloated.

No he  misunderstood and so did you because I said I would  nuke it.

**Comment by seiyria** [comment]
@Nizzypal 

No one is ganging up on you. We're neither being mean nor discriminatory.  We're acting like adults. However, your  response is completely unacceptable. Keep tickets on topic and keep this behaviour out of it.

**Review by seiyria — APPROVED** [review]

**Pull request merged**